lib/tasks/projects.rake

```diff
@@ -23,7 +23,7 @@ namespace :projects do
   desc 'Check status of projects'
   task check_status: :environment do
     exit if ENV['READ_ONLY'].present?
-    ['npm', 'rubygems', 'packagist', 'nuget', 'cpan', 'clojars', 'cocoapods',
+    ['pypi', 'npm', 'rubygems', 'packagist', 'cpan', 'clojars', 'cocoapods',
```

Comment:
Changed the list of package managers to match those in `projects:check_removed_status`, which was updated recently.